src/services/shopify-client.doubles.ts

```diff
@@ -44,7 +44,16 @@ export class HardcodedShopifyClient implements ShopifyClient {
     await timeout(1500);
     //return { success: false, message: 'Some random error' }; // Unexpected error
     //return { success: true, value: twoShops }; // Shopify connected with more than one shop
-    return { success: true, value: oneShop }; // Shopify connected to one shop
     //return { success: true, value: [] }; // Shopify disconnected
+    return { success: true, value: oneShop }; // Shopify connected to one shop
+    /*
+    //Test state change behaviour
+    if (new Date().getTime() % 2 !== 0) {
+      oneShop[0].list.amountSubscribers++;
+      return { success: true, value: oneShop };
+    } else {
+      return { success: true, value: [] }; 
+    }
+    */
```

Comment:
It is very arguable, so, please, take it as a simple suggestion.

What do you think about using _line comments_ (`// comment...`), in place of _block comments_ ( `/* comment ... */`).

They are more friendly with VS Code, you can use `CTRL+K CTRL+C` to comment and `CTRL+K CTRL+U` to uncomment.

Reply:
Fixed. `Ctrl + / ` in vs Code to toggle inline comments.